docs/Concepts/7.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Security Features
+---
+
+Each protocol layer of Waku v2 provides a distinct service and is associated with a separate set of security features and concerns.
+Therefore, the overall security of Waku v2 depends on how the different layers are utilized.
+
+The security models are detailed in the RFC of the protocols.
+That we strive to provide well documented and open source RFCs so that Waku users know what security guarantees are and aren't provided by each protocol.
```

Comment:
```suggestion
We strive to provide well documented and open source RFCs so that Waku users know what security guarantees are and aren't provided by each protocol.
```